packages/engine-test-utils/src/__tests__/common-all/util/treeUtil.spec.ts

```diff
@@ -3,76 +3,76 @@ import _ from "lodash";
 import { runEngineTestV5 } from "../../../engine";
 import { ENGINE_HOOKS } from "../../../presets";
 
-describe("WHEN regular tree", () => {
-  test("THEN return regular tree", async () => {
-    await runEngineTestV5(
-      async ({ engine }) => {
-        const treeData = TreeUtils.generateTreeData(engine.notes, [
-          engine.notes["foo"],
-        ]);
-        expect(treeData.roots).toMatchSnapshot();
-      },
-      {
-        expect,
-        preSetupHook: ENGINE_HOOKS.setupBasic,
-      }
-    );
-  });
-});
-
-describe("WHEN nav_exclude_children enabled", () => {
-  test("THEN return only root", async () => {
-    await runEngineTestV5(
-      async ({ engine }) => {
-        const domain = engine.notes["foo"];
-        domain.custom.nav_exclude_children = true;
-        const treeData = TreeUtils.generateTreeData(engine.notes, [domain]);
-        expect(treeData.roots).toMatchSnapshot();
-        expect(treeData.roots[0].children).toEqual([]);
-      },
-      {
-        expect,
-        preSetupHook: ENGINE_HOOKS.setupBasic,
-      }
-    );
-  });
-});
-
-describe("WHEN has_collection enabled", () => {
-  test("THEN return only root", async () => {
-    await runEngineTestV5(
-      async ({ engine }) => {
-        const domain = engine.notes["foo"];
-        domain.custom.nav_exclude_children = true;
-        const treeData = TreeUtils.generateTreeData(engine.notes, [domain]);
-        expect(treeData.roots).toMatchSnapshot();
-        expect(treeData.roots[0].children).toEqual([]);
-      },
-      {
-        expect,
-        preSetupHook: ENGINE_HOOKS.setupBasic,
-      }
-    );
-  });
-});
-
-describe("WHEN has_collection enabled AND nav_children_exclude set to false", () => {
-  test("THEN return only root", async () => {
-    await runEngineTestV5(
-      async ({ engine }) => {
-        const domain = engine.notes["foo"];
-        domain.custom.nav_exclude_children = true;
-        const treeData = TreeUtils.generateTreeData(engine.notes, [domain]);
-        expect(treeData.roots).toMatchSnapshot();
-        expect(treeData.roots[0].children).toEqual([]);
-      },
-      {
-        expect,
-        preSetupHook: ENGINE_HOOKS.setupBasic,
-      }
-    );
-  });
-});
+// describe("WHEN regular tree", () => {
```

Comment:
are we adding these tests back in?

Reply:
yes, done here [`e1b192c` (#3448)](https://github.com/dendronhq/dendron/pull/3448/commits/e1b192c0a64aba6cd6e7d1050c4441b4e406a2b1)
and little correction here [`4a37d82` (#3448)](https://github.com/dendronhq/dendron/pull/3448/commits/4a37d82fb42191279fd326f05507b2e8ba540671)